### PR TITLE
Presence update: skindb.co

### PR DESCRIPTION
**Pull request by NotOfficer**
Added support for [skindb.co/fallguys](https://skindb.co/fallguys).
Proof: [media.wtf/95781050](https://media.wtf/95781050)

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues.

**Comment by NotOfficer** [comment]
For real...?

**Comment by NotOfficer** [comment]
I dont even get what it wants me to change.

File: websites/S/skindb.co/presence.ts [hunk 1]
```diff
@@ -87,6 +87,31 @@ presence.on("UpdateData", async () => {
...
+      path.startsWith("/bottom") ||
+      path.startsWith("/pattern") ||
+      path.startsWith("/emote") ||
+      path.startsWith("/faceplate") ||
+      path.startsWith("/colour") ||
+      path.startsWith("/celebration")
+    ) {
+      const skinName = document.querySelector(".skin-name");
+      const skinType = document.querySelector(".skin-type-string");
+      const skinRarity = document.querySelector(".skin-rarity-string > .rarity-label");
```
Comment (Bas950) [inline websites/S/skindb.co/presence.ts:111]: like merge these consts, so like:
```
const var = val,
 var2 = val,
 var3 = val;
```

Comment (NotOfficer) [reply websites/S/skindb.co/presence.ts]: thanks

**Review by Bas950 — APPROVED** [review]

**Review by doomlerd — APPROVED** [review]

**Pull request merged**